### ISSUE TITLE
Trying to get your fix working on Python3

### ISSUE THREAD

**Issue by robboz4** [mamacker/pi_to_potter#1]
HI!

Thanks for addressing the error in the original Make article. However, I'm trying to get the code to run on Python3. I followed the original article setting it up all on Python3. However, python3 on my Pi is very picky about tabs versus spaces on indentation. First I added "()" around the print statements and indented them where obvious. However, I'm not sure about line 174 - the while statement. I have indented it with two tabs under noPT=0, but I still get 
sudo python3 rpotter.py 
  File "rpotter.py", line 174
    while True:
              ^
TabError: inconsistent use of tabs and spaces in indentation

Is the while loop part of the track wand  function or is  the main loop of code?

Thanks in advance...

**Comment by robboz4** [comment]
Never mind!

I hadn't reloaded the file to the Pi after I made the changes. It seems to be running now. There are lot of errors at the beginning, from OpenCV. As I'm in the test and build phase I pointed a TV remote at  the camera and the errors disappear and the print statements from  the code came out. So on to the next part making a wand and wiring up the IR Led and neopixels and relays. And thanks once again for fixing the issue that was tripping me up base don the article.
